virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -538,3 +538,35 @@ def create_remote_disk_by_same_metadata(vm, params):
 
     remote_session.close()
     return blk_source
+
+
+def fill_sth_in_vm(vm, target, size_value=500):
+    """
+    File sth in the disk of VM
+
+    :param vm: VM guest
+    :param target: disk dev in VM
+    :param size_value: number in MiB
+    """
+    try:
+        session = vm.wait_for_login()
+        rpm_stat, out_put = session.cmd_status_output("rpm -q parted || "
```

Comment:
I would suggest to use package_install('parted') method from the virttest/utils_package instead of doing the same manually.

Reply:
updated